veomni/data/multimodal/video_utils.py

```diff
@@ -216,3 +269,35 @@ def fetch_videos(videos: List[VideoInput], **kwargs):
         smart_audio_nframes(audio, audio_fps, **kwargs) for audio, audio_fps in zip(audio_inputs, audio_fps_list)
     ]
     return video_inputs, audio_inputs
+
+def fetch_videos_metadata(videos: List[VideoInput], **kwargs):
```

Comment:
Can we merge this function into `fetch_videos`? It seems return metadata will be necessary, so `fetch_videos` returns videos, video_metadata, audios, audio_metadata